### PR TITLE
Vulnerability fix

### PR DESCRIPTION
**Pull request by edanfesi**
## What does this PR do?

Remove teamid from query param

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?
- [x] Remove the teamId from the query param and you still be able to create event type for that team

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/components/eventtype/CreateEventType.tsx [hunk 1]
```diff
@@ -51,9 +51,7 @@ export default function CreateEventTypeButton(props: Props) {
 
   // URL encoded params
   const teamId: number | undefined =
```
Comment (agustif) [inline apps/web/components/eventtype/CreateEventType.tsx:53]: Why leave 53 with teamId if we're no longer using it? It's initialized as const as empty?

**Review by agustif — CHANGES_REQUESTED** [review]
Hi @edanfesi, I think we need to find a better way to fix this, as by removing the teamId props and passing it the id it breaks when you create a new event type for a team for me.


Reproduce:
Try to create a new event under a team:

Screenshot 2022-02-15 at 16 43 59

Scheduling settings won't show due to the new bug.
image

It should appear like this:
image



PS: I think you added a lot of edits from prettier/eslint fixes in unrelated files, would be good if you can get rid of those so your commits are only about this change.

File: apps/web/prisma/zod/attendee.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import * as z from "zod";
```
Comment (zomars) [inline apps/web/prisma/zod/attendee.ts:1]: Prettier should ignore these auto-generated files. Everything in ` apps/web/prisma/zod` basically.

**Review by zomars — CHANGES_REQUESTED** [review]
Adding on the comments from @agustif I don't think just removing the `teamId` from query is an optimal solution. We can leave the query. Let's discuss the sensitive details on Slack.

File: apps/web/components/eventtype/CreateEventType.tsx [hunk 3]
```diff
@@ -98,7 +101,6 @@ export default function CreateEventTypeButton(props: Props) {
             ...router.query,
             new: "1",
             eventPage: option.slug,
-            teamId: option.teamId || undefined,
```
Comment (zomars) [inline apps/web/components/eventtype/CreateEventType.tsx:101]: This is preventing me to create team events

**Review by zomars — CHANGES_REQUESTED** [review]

File: apps/web/components/eventtype/CreateEventType.tsx [hunk 4]
```diff
@@ -52,9 +52,7 @@ export default function CreateEventTypeButton(props: Props) {
 
   // URL encoded params
   const teamId: number | undefined =
-    typeof router.query.teamId === "string" && router.query.teamId
-      ? parseInt(router.query.teamId)
-      : undefined;
```
Comment (zomars) [inline apps/web/components/eventtype/CreateEventType.tsx:57]: Let's revert this change as well

**Comment by zomars** [comment]
Closed in favor of #1866
